src/main/kotlin/org/jitsi/nlj/rtp/TransportCcEngine.kt

```diff
@@ -103,7 +103,7 @@ class TransportCcEngine(
 
     private fun tccReceived(tccPacket: RtcpFbTccPacket) {
         val now = clock.instant()
-        var currArrivalTimestamp = Instant.ofEpochMilli(tccPacket.GetBaseTimeUs() / 1000)
+        var currArrivalTimestamp = Instant.ofEpochMilli((tccPacket.GetBaseTimeUs() + 500) / 1000)
```

Comment:
Dumb question: This is effectively rounding up instead of rounding down, is that right?  If so, is it somewhat arbitrary if we do one or the other?

Reply:
No, integer division truncates, so this rounds to the nearest integer. E.g.
```
600 / 1000 = 0
(600 + 500) / 1000 = 1
```

Reply:
🤦‍♂ 👍